notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,484 @@
+{
```

Comment:
<p>I would delete the second sentence - I think users should be going through sequentially and not hopping around tutorial</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I think I'm in agreement with @WillAyd's suggestion here. Perhaps don't encourage users to jump ahead to future lessons, though in future lessons it does make sense to link back to previous lessons.</p>

Reply:
<p>ok, good point!</p>